lib/index.js

```diff
@@ -151,6 +152,7 @@ class NewsArticle extends BaseAsset {
     to_metadata() {
         const metadata = super.to_metadata();
         metadata['document'] = this._document.toString();
+        metadata['synopsis'] = this._synopsis.toString();
```

Comment:
@syndacks Yes, that's the exact problem I'm talking about. Without a document, a news asset is pretty much not working properly to begin with so it's fine if errors happen there but for synopsis, if there isn't one, you can make sure that you don't set one on the metadata synopsis.

PS: I know it might be a bit much to ask right now but there are some tests that you can update to make sure this code works as you expect it: https://github.com/endlessm/libingester/blob/master/test/lib/index.test.js

You can run the tests with: https://github.com/endlessm/libingester/blob/master/run_tests.sh